trinity/utils/chains.py

```diff
@@ -81,6 +81,21 @@ def get_database_socket_path(chain_identifier: str, data_dir: str=None) -> str:
     )
 
 
+JSONRPC_SOCKET_FILENAME = 'jsonrpc.ipc'
+
+
+def get_jsonrpc_socket_path(chain_identifier, data_dir=None):
```

Comment:
There are some method signatures in this PR like this one that are missing type hints. I guess we want all new Trinity code to land as typed right from the start.

Reply:
I originally did this PR including the addition of type hints, but I had a bunch of hard-to-debug failures and I reverted everything to the minimal implementation that worked.  I'm going to do type hints for this stuff right away after this is merged (assuming it passes CI).

Reply:
:+1: